docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,821 @@
+     - Total prevalence of opioid use disorder
+     - Includes both treated and untreated OUD
+   * - deaths_c562
+     - GBD 2023 (CoDCorrect)
+     - Deaths attributed to opioid use disorder
+     - Direct OUD deaths; excludes opioid overdose deaths coded separately
+   * - population
+     - GBD 2023 (Demography)
+     - Mid-year population by age/sex/location/year
+     - Standard GBD population estimates
+   * - treatment_coverage
+     - ST-GPR model of percentage of dependents in treatment (if we can find it!)
+     - Proportion of individuals with OUD receiving MOUD
+     - This covariate is mentioned in the description of the IHME-indirect data creation in the Supplementary Appendix section on OUD
+   * - Transition rates (i, r, ti, td, ts)
+     - NumPyro DisMod-AT-like model
+     - Internally consistent transition rates
+     - Estimated using Bayesian model that integrates GBD prevalence, treatment coverage, and other available data to solve for consistent set of transition rates
+   * - emr_base
+     - GBD 2023 (DisMod-MR 2.1)
+     - Base excess mortality rate for OUD
+     - :math:`\text{EMR} = \frac{\text{CSMR}_{\text{c562}}}{\text{prevalence}_{\text{c562}}}`
+   * - Disability weights
+     - GBD 2023 (YLD Appendix)
+     - Disability weights for OUD sequelae
+     - Asymptomatic (0), mild (0.335), moderate/severe (0.697)
+   * - sequelae_c562
+     - GBD 2023 (GBD Mapping)
+     - Sequelae for opioid use disorder
+     - Asymptomatic, mild, and moderate/severe opioid dependence
+
+Estimation of Transition Rates Using NumPyro/DisMod-AT
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A key challenge in parameterizing this model is that GBD 2023 does not directly provide all required transition rates (particularly natural remission *r*, treatment initiation *ti*, treatment discontinuation *td*, and treatment-associated recovery *ts*). To address this, we use a **NumPyro implementation of a DisMod-AT-like model**.
+
+**Methodology**
+
+DisMod-AT (Disease Model – Age-and-Time) is a Bayesian meta-analytic tool designed to synthesize diverse epidemiological data to produce internally consistent transition rates for compartmental disease models. Our NumPyro implementation:
+
+1. **Model Structure**: Configures a three-state compartmental model (Susceptible, With Condition, On Treatment) matching the structure defined above
+
+2. **Input Data**: Incorporates:
+
+   - GBD 2023 prevalence estimates for OUD
+   - Treatment coverage ratios (proportion of OUD cases receiving MOUD)
+   - GBD 2023 incidence estimates (as prior/constraint)
+   - Excess mortality estimates from GBD 2023
+   - Literature-based estimates or assumptions about remission and treatment rates
+
+3. **Bayesian Inference**: Uses Markov Chain Monte Carlo (MCMC) sampling to estimate the posterior distribution of all transition rates conditional on:
+
+   - Observed prevalence matching GBD estimates
+   - Treatment coverage matching available data
```

Comment:
This list should include the treatment retention/discontinuation studies. Also, as I noted above, it seems a stretch to assume discontinuation = return to active addiction. This needs clarification/noting.